pallets/subtensor/tests/benchmarking.rs

```diff
@@ -1,3 +1,4 @@
+
```

Comment:
needed extra line?

Reply:
no :D I cleaned up but this line was left out.